src/index.ts

```diff
@@ -87,25 +87,23 @@ function flourite(
   for (let i = 0; i < pairs.length; i++) {
     const { language, checkers } = pairs[i];
 
-    const pointsList: number[] = [];
+    let points = 0;
     for (let j = 0; j < linesOfCode.length; j++) {
+      // fast return if the current line of code is empty or contains only spaces
+      if (linesOfCode[j].match(/^\s*$/)) {
```

Comment:
`.test` might be more suitable because it returns a boolean.

Reply:
ok